ReText/window.py

```diff
@@ -211,8 +211,8 @@ def __init__(self, parent=None):
 			trig=lambda: self.insertChars('*'))
 		self.actionUnderline = self.act(self.tr('Underline'), shct=QKeySequence.Underline,
 			trig=lambda: self.insertTag('u'))
-		self.usefulTags = ('a', 'big', 'center', 'img', 's', 'small', 'span',
-			'table', 'td', 'tr', 'u')
+		self.usefulTags = ('header', 'italic', 'bold', 'numbering', 'bullets', 'image', 'link',
```

Comment:
I don't know why it wouldn't work.

Keeping both actions and toolbar items is also fine, just maybe we can make them use the shared code?

My priority now is fixing an XSettings crash on Xfce and getting a bug fix release out, after that I will merge your pull request.